Helpers/JsonLD.cs

```diff
@@ -8,24 +8,37 @@ namespace Sedos.Helpers
 {
     public class JsonLD
     {
+        private static Organization _sedosOrganization = new()
```

Comment:
```suggestion
        private static Organization SedosOrganization = new()
```

I really dislike the `_` convention :D